.github/workflows/test.yml

```diff
@@ -26,7 +26,9 @@ jobs:
         check-latest: true
 
     - name: Install dependencies for x_benthos_extra
-      run: sudo apt install -y --no-install-recommends libzmq3-dev
+      run: |
+        sudo apt install -y --no-install-recommends libzmq3-dev
+        go generate -tags=x_benthos_extra ./internal/impl/ollama
```

Comment:
Yes done.

Reply:
I undid this change, redownloading this every `make docs` seems broken, so I've made it more explicit. Maybe there is makefile magic that can prevent running this if the file is already there? Honestly I'm pretty terrible at raw make. Let me know if there a strong preference here!

Reply:
I cobbled something together - PTAL I'm not good with Makefiles 😄

Reply:
Nice 🥇 That's what I was expecting to work. Thanks for digging into it! I think it's probably not that relevant to have both `ollama-linux-amd64` and `ollama-linux-arm64` or they could be placed in a folder called `ollama` or `ollama-resources` or smth like that so then you won't really need that `if` statement in the Makefile because you can check for the generic name. Guess you can remove the `go generate` stuff from the CGo Dockerfile.

Reply:
Just named them both `ollama` that cleaned things up thanks!

Reply:
You're welcome! I think you can actually use `./internal/impl/ollama/ollama` as the dependency of the target and then you don't even need the if inside, but that's just a nitpick :)

Reply:
Hrm but it's not working in CI as it can't find the file to embed - it seems to work locally... Any tips?

Reply:
So this is what I had in mind:
```
./internal/impl/ollama/ollama:
	go generate -tags "$(TAGS)" ./internal/impl/ollama

generate: ./internal/impl/ollama/ollama
```
And then you'd need to make sure `generate` is called first for the `$(APPS)` target like so:

```
$(APPS): %: generate $(PATHINSTBIN)/%
```

That should do the trick I think. If you get stuck let me know and I'll send you the full code.

Reply:
Really appreciate that! That seems to work for me locally, let's see how CI feels about it.

Reply:
Seems to work - thank you! 🙏

Reply:
You're welcome!